### ISSUE TITLE
Multistore with store codes

### ISSUE THREAD

**Issue by AdorableTheodore** [elgentos/Mailcheck#5]
This module is not working with store codes in URLs. I have not investigated how to fix this, I have only got as far as installing this wonderful looking extension and then wondering why it does not do anything - turns out that the multi store store codes were the problem.

**Comment by peterjaap** [comment]
Fixed; https://github.com/elgentos/Mailcheck/commit/d2ae4b2a104a05638fc2f63dbaa8c100c7ecd56a

Thanks for notifying :)

**Comment by AdorableTheodore** [comment]
Thanks! I gave it a quick test but it looks like I need to do a bit more testing - same as before - but I will try on my demo store first...
I log anything that does a Magento 404 (rather than apache 404) as we recently moved to multi store and a lot of modules get caught out.
I will let you know what I have done wrong :-) Could be just unpacking the extension!!!
Thanks again,
Mathew 

```
 On Tuesday, 6 October 2015, 15:02, Peter Jaap Blaakmeer <notifications@github.com> wrote:
```

 Closed #5.—
